.github/workflows/testing.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
             ${{ runner.os }}-
-      - run: python -m pip install pre-commit
+      - run: python -m pip install pre-commit pylint
```

Comment:
Don't think this should be required. Pre-commit should handle all the package installs on its own, like it does with flake8, black, etc.

Was it failing?